frontend/src/pages/projects/screens/detail/pipelines/PipelinesSection.tsx

```diff
@@ -40,25 +51,43 @@ const PipelinesSection: React.FC = () => {
   );
 
   return (
-    <PipelineAndVersionContextProvider>
+    <>
```

Comment:
Why did we drop this? Did you move it to another layer? I can only see two use-cases and those are not in Projects 🤔